lib/reward.ts

```diff
@@ -0,0 +1,71 @@
+import type { Address, PublicClient } from 'viem'
+import { formatEther, encodeAbiParameters} from 'viem';
+import proxyAbi from '@/lib/proxy-abi';
+import { PROXY_CONTRACT_ADDRESS } from '@/lib/config';
+
+
+async function getRewarderAddress(
+  attestationType: string,
+  client: PublicClient,
+): Promise<Address> {
+  const [,rewarderAddress,] = await client.readContract({
+    address: PROXY_CONTRACT_ADDRESS,
+    abi: proxyAbi,
+    functionName: 'attestationTypes',
+    args: [attestationType],
+  }) as [any, Address, any];
```

Comment:
is this cast necessary? proxyAbi is imported from a const ABI, so it should be able to infer here